src/TestWebsites/MVC/config/imageprocessor/cache.config

```diff
@@ -1,6 +1,6 @@
 ﻿<caching currentCache="DiskCache">
   <caches>
-    <cache name="DiskCache" type="ImageProcessor.Web.Caching.DiskCache, ImageProcessor.Web" maxDays="365">
+    <cache name="DiskCache" type="ImageProcessor.Web.Caching.DiskCache, ImageProcessor.Web" maxDays="365" browserMaxDays="365">
```

Comment:
I wonder if we should make this work so if the element is not present, it defaults to using maxDays for compatibility?